minard/views.py

```diff
@@ -1576,3 +1579,69 @@ def _dropout_detail_n100(run_number):
 @app.route("/dropout/_dropout_detail/N20/<int:run_number>")
 def _dropout_detail_n20(run_number):
     return dropout.get_details(run_number, 2)
+
+@app.route('/standard_runs/<uuid>', methods=['GET', 'POST'])
+@app.route("/standard_runs/")
+@app.route("/standard_runs")
+def standard_runs(uuid=None):
+    if uuid is None:
+        values = None
+        try:
+            values = sr.get_standard_runs()
+        except sr.couchdb.http.socket.error:
+            flash("Error connecting to database, could not retrieve standard runs", "danger")
+        return render_template('standard_runs.html', values=values)
+    if request.method =='POST':
+        updater = request.form.get("name")
+        info = request.form.get("info")
+        passw = request.form.get("password")
+        new_values = [x[:-8] for x in request.form.keys() if x[-8:] == "_replace"]
+        new_values = {x:request.form[x+"_value"] for x in new_values}
```

Comment:
The python running on minard is python 2.6 which unfortunately doesn't support this syntax for creating dictionaries.